targets/xorg

```diff
@@ -68,7 +68,7 @@ if [ "${ARCH#arm}" = "$ARCH" ]; then
 fi
 
 # Catalog relevant and irrelevant video drivers
-fbdev="xserver-xorg-video-fbdev$backport"
+fbdev="openSUSE=xf86-video-fbdev,xserver-xorg-video-fbdev$backport"
```

Comment:
I don't see any schema that would be easy to implement. Some have a prefix, some have a postfix and some have a different name.